### PR TITLE
Update test

### PR DESCRIPTION
**Pull request by vorian**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/vorian/test/pull/2" target="_blank">ReviewNinja

File: test [hunk 1]
```diff
@@ -1 +1,2 @@
 test
+some change
```
Comment (vorian-alt) [inline test]: !fix 
some problem

File: test [hunk 2]
```diff
@@ -1 +1,2 @@
 test
```
Comment (vorian) [inline test]: !fixed

Comment (vorian) [inline test]: !fix

File: test (hunk 1 above)
Comment (vorian-alt) [inline test]: !fixed

File: test (hunk 2 above)
Comment (vorian-alt) [inline test]: !fixed

**Pull request merged**